### PR TITLE
Add doc for helm CLI 3.7

### PR DESCRIPTION
**Pull request by YangJiao0817**
Signed-off-by: Yang Jiao <jiaoya@vmware.com>

### CODE REVIEW

File: docs/working-with-projects/working-with-images/managing-helm-charts.md [hunk 1]
```diff
@@ -155,6 +155,7 @@ For other more helm commands like how to sign a chart, please refer to the [helm
 
 ## Manage Helm Charts with the OCI-compatible registry of Harbor
 
+### If the version of Helm CLI you are using is before 3.7.0, please refer to the following
```
Comment (OrlinVasilev) [inline docs/working-with-projects/working-with-images/managing-helm-charts.md:158]:
```suggestion
### If the Helm CLI version is lower than 3.7.0, please refer to the following
```

File: docs/working-with-projects/working-with-images/managing-helm-charts.md [hunk 2]
```diff
@@ -197,6 +198,49 @@ To pull charts from the the OCI-compatible registry of Harbor, run the `helm cha
 ```sh
 helm chart pull xx.xx.xx.xx/library/dummy-chart:version
 ```
+### If the version of Helm CLI you are using is 3.7.0 or later, please refer to the following
```
Comment (OrlinVasilev) [inline docs/working-with-projects/working-with-images/managing-helm-charts.md:201]:
```suggestion
### If the Helm CLI version is 3.7.0 or higher, please refer to the following
```

File: docs/working-with-projects/working-with-images/managing-helm-charts.md [hunk 3]
```diff
@@ -197,6 +198,49 @@ To pull charts from the the OCI-compatible registry of Harbor, run the `helm cha
...
+
+```sh
+helm version
+
+#version.BuildInfo{Version:"v3.7.0", GitCommit:"eeac83883cb4014fe60267ec6373570374ce770b", GitTreeState:"clean", GoVersion:"go1.16.8"}
+```
+
+### Login to the OCI-compatible registry of Harbor
+
+Before pull/push helm charts with the OCI-compatible registry of Harbor, Harbor should be logged with `helm registry login` command.
```
Comment (OrlinVasilev) [inline docs/working-with-projects/working-with-images/managing-helm-charts.md:212]:
```suggestion
Prior pull/push helm charts with the OCI-compatible registry of Harbor, should "login" by `helm registry login` command.
```

**Comment by OrlinVasilev** [comment]
@YangJiao0817 @a-mccarthy made some suggestions :)

**Comment by YangJiao0817** [comment]
Thanks @OrlinVasilev ,has been modified.

**Review by OrlinVasilev — APPROVED** [review]
LGTM

**Comment by OrlinVasilev** [comment]
@a-mccarthy can you check as well please!

**Comment by OrlinVasilev** [comment]
@YangJiao0817 is that still valid or needs some update?

**Comment by OrlinVasilev** [comment]
merged! thank you!

**Pull request merged**